python/ray/data/tests/test_resource_manager.py

```diff
@@ -624,6 +624,95 @@ def test_only_handle_eligible_ops(self, restore_data_context):
         allocator.update_usages()
         assert o2 not in allocator._op_budgets
 
+    def test_gpu_allocation(self, restore_data_context):
+        """Test GPU allocation for GPU vs non-GPU operators."""
+        DataContext.get_current().op_resource_reservation_enabled = True
+        DataContext.get_current().op_resource_reservation_ratio = 0.5
```

Comment:
Here and for the test below -- aren't these the defaults? Are they necessary for this test?